src/test/javascript/spec/directive/sidebar-card.directive.spec.ts

```diff
@@ -64,10 +64,6 @@ describe('SidebarCardDirective', () => {
         jest.resetAllMocks();
     });
 
-    it('should create an instance', () => {
```

Comment:
Seems like the template test for me, I do not think that we need it

The next test checks this aswell anyways